packages/commonwealth/client/scripts/views/components/MetaTags/MetaTags.tsx

```diff
@@ -81,7 +82,7 @@ const defaultMeta = {
   },
   'twitter:image': {
     name: 'twitter:image',
-    content: 'https://commonwealth.im/static/img/branding/common.png',
+    content: commonUrl,
```

Comment:
this should be a full url, including the domain part.

Reply:
I will move this file into the public folder so that it doesn't change. But this path will change a little bit in the PR because it we will no longer serve static assets on the /static/ path.